kf_lib_data_ingest/etl/load/load_v1.py

```diff
@@ -87,11 +87,13 @@ def __init__(
 
         target = urlparse(target_url).netloc or urlparse(target_url).path
         self.uid_cache_filepath = os.path.join(
-            cache_dir or os.getcwd(),
```

Comment:
why remove this?

Reply:
Wellll, bc when you instantiate loader with cache_dir=None, it throws an exception bc the base Stage class uses it

Reply:
Oh also, its not removed. I just moved setting cache_dir up in the constructor

Reply:
but it looks like this doesn't use the cache dir anymore

Reply:
Ohhh yea sorry I see what you're saying now. I accidentally removed that little important line altogether.